modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -6,55 +6,50 @@
 [id="nw-ingress-converting-http-header-case_{context}"]
 = Converting HTTP header case
 
-HAProxy lowercases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
+HAProxy lowercases HTTP header names by default; for example, changing `Host: xyz.com` to `host: xyz.com`. If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
 
 [IMPORTANT]
 ====
-Because {product-title} includes HAProxy 2.8, be sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading.
+{product-title} includes HAProxy 2.8. If you want to upgrade to this version of the web-based load balancer, ensure that add `spec.httpHeaders.headerNameCaseAdjustments` to your cluster's configuration file.
```

Comment:
I think the phrase "ensure that add `spec.httpHeaders.headerNameCaseAdjustments` to your cluster's configuration file." is missing something or needs to be reworked, could you take a look and readjust depending on the intended meaning?

Reply:
Updated to state:

> ensure that you add the `spec.httpHeaders.headerNameCaseAdjustments` section to your cluster's configuration file.